src/backend/InvenTree/plugin/builtin/labels/inventree_machine.py

```diff
@@ -112,8 +112,9 @@ def __init__(self, *args, **kwargs):
             super().__init__(*args, **kwargs)
 
             view = kwargs['context']['view']
-            template = view.get_object()
-            items_to_print = view.get_items()
+            # TODO @matmair Re-enable this when the need is clear
+            template = None  # view.get_object()
+            items_to_print = None  # view.get_items()
```

Comment:
The idea is to give printing drivers the ability to decide wherever a printer would be able to print this job. E.g. 
- A printer which has only 10x10mm labels cannot print large 30x70mm labels, therefore it wouldn't make sense to show that printer to the user for selection 
*(plugins would need to add a settings field where one could enter the currently available label sizes in the printer, and then override the [`get_printers`](https://docs.inventree.org/en/stable/extend/machines/label_printer/#machine.machine_types.LabelPrinterBaseDriver.get_printers) function to then filter for printers that could print that size of template)*. 
- Same for the actual items, maybe it's not that likely to print with a printer in a different factory when printing a location label. But since it could happen, I have decided to let this one open for future implementation.

Reply:
@SchrodingersGat @matmair What happend with this?

Reply:
I do not think this was adressed